### PR TITLE
Extract the StanfordClient to a class

### PR DESCRIPTION
**Pull request by jcoyne**
This favors aggregation over inheritance

### CODE REVIEW

File: lib/rialto/etl/cli/extract.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def call(name)
         desc 'list', 'List callable extractors'
         # List callable extractors
         def list
-          callable_extractors = Rialto::Etl::Extractors.constants.reject { |name| name.to_s.include?('Abstract') }
+          callable_extractors = Rialto::Etl::Extractors.constants.map(&:to_s) - ['StanfordClient']
```
Comment (mjgiarlo) [inline lib/rialto/etl/cli/extract.rb]: Now that we don't need to do a substring check against the names of the extractors, I don't believe we need to coerce to a string here. How would you feel about changing this method to look like:

```ruby
callable_extractors = Rialto::Etl::Extractors.constants - [StanfordClient] # or .constants.reject { |name| name == StanfordClient }
say "Extractors supported: #{callable_extractors.join(', ')}"
```

:question:

Comment (mjgiarlo) [reply lib/rialto/etl/cli/extract.rb]: Alternatively, we could move the Stanford client out of the `extractors` dir into a new area for clients?

File: spec/extractors/stanford_client_spec.rb [hunk 2]
```diff
@@ -90,13 +90,11 @@
...
 
-  describe '#extract' do
+  describe '#get' do
     subject(:extractor) { described_class.new }
 
-    it { is_expected.to respond_to(:extract) }
-
     it 'does not raise NotImplementedError' do
-      expect { extractor.extract }.to raise_error(NotImplementedError)
+      expect(extractor.get('/path')).to eq 'whatever'
```
Comment (mjgiarlo) [inline spec/extractors/stanford_client_spec.rb]: Where's the `'whatever'` coming from?

Comment (jcoyne) [reply spec/extractors/stanford_client_spec.rb]: line 8

File: lib/rialto/etl/cli/extract.rb (hunk 1 above)
Comment (mjgiarlo) [reply lib/rialto/etl/cli/extract.rb]: I'm cool with that, too.

**Pull request merged**